rust/tw_coin_registry/src/dispatcher.rs

```diff
@@ -41,6 +43,7 @@ const INTERNET_COMPUTER: InternetComputerEntry = InternetComputerEntry;
 const NATIVE_EVMOS: NativeEvmosEntry = NativeEvmosEntry;
 const NATIVE_INJECTIVE: NativeInjectiveEntry = NativeInjectiveEntry;
 const PACTUS: PactusEntry = PactusEntry;
+const POLKADOT: SubstrateEntry<PolkadotEntry> = SubstrateEntry(PolkadotEntry);
```

Comment:
Could you please add a type alias `PolkadotEntry = SubstrateEntry<PolkadotEntryImpl>` or something like that in `tw_polkadot` crate, so we have the same way here to declare blockchains.

Reply:
Even with the type alias we can't do `const POLKADOT: PolkadotEntry = PolkadotEntry`, since we can't create the type from the alias.

The best I can do is:
```rust
const POLKADOT: PolkadotEntry = SubstrateEntry(PolkadotEntryImpl);
```

Reply:
Right, haven't predicted that. Let's leave it for now